release-team/role-handbooks/enhancements/README.md

```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
     - Until this can be automated; manually create Views with the same Names and Fields from the previous release's project board.
       Tip: You can easily view all the Fields present in each view by clicking the `v` next to the View's name an selecting `Configuration` -> `Fields`
       Note: Remember to **save** each View. If there is a blue bubble next to the Views name there are unsaved changed for that view!
+  - Update automation to add issues to the correct Enhancement Tracking Board
+    - TODO: Work in Progress
```

Comment:
whats missing to describe the sub steps?

Reply:
We need to get it working first 🤣
AFAIK we are still having issues with the PROW job and hadn't decided if we were going to continue to try to run this in PROW or as a GitHub Action.

Reply:
👍 ok